citation/globals.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
```

Comment:
How about adding a `graphviz.py` module and putting the contents of this globals and the various `generate_*` methods there?

Reply:
I'm creating a graphviz directory in citation and adding content according in viz_views_helper.py and globals.py file. Cannot move all content (globals and views part) in one module because it creates an import deadlock. let me know if you any other suggestion.